miden/src/examples/mod.rs

```diff
@@ -100,7 +100,7 @@ impl ExampleOptions {
         // serialize the proof to see how big it is
         let proof_bytes = proof.to_bytes();
         println!("Execution proof size: {} KB", proof_bytes.len() / 1024);
-        println!("Execution proof security: {} bits", proof.security_level(true));
+        println!("Execution proof security: {} bits", proof.security_level::<Rpo256>(true));
```

Comment:
We should still be using `BLAKE3` here.

We also need to pick the right version of `BLAKE3` depending on target security level (i.e., 256-bit vs. 192-bit). This might require tracking extra info in the `ProofObject`. But we can do this in a subsequent PR.

Reply:
Issue added: #710